### ISSUE TITLE
Combining Data

### ISSUE THREAD

**Issue by brittanyukulele** [orzubalsky/tradeschool#65]
Is there a way to combine data in the new system? I am seeing the same teachers in the new database, but it's giving me two of them because we must have used different email addresses.

**Comment by orzubalsky** [comment]
Yes, that would be great! I've been thinking of ways to do it too, it's not trivial. I'll update when it becomes more concrete. In the meantime I'm going to focus on fixing bugs, since this feels like a lower priority.

**Comment by ghost** [comment]
This issue was moved to Z1800000000/tradeschool#30
